dashboard2/publishwebsite.md

```diff
@@ -0,0 +1,154 @@
+Publish website
+===============
+
+In this menu, you are provided with the means of publishing your website on a pretty domain after translation tasks have finished.
+
+The publish wizard has two options to make the site available to the web at large: serving domain mode and subdirectory publishing mode.
+
+
+## Domain settings
+
+This screen opens by default after clicking on a _Target locale_ in your list. The following settings and fields are available:
+
+### Access control
+
+You can add basic HTTP authentication to the proxied site in order to restrict access to it. Select the proxy modes that you'd like to limit access to, add a user by entering a username and password. You can add multiple users to the list.
+
+### Publish website wizard
+
+Under this menu, you will be able to publish your website either in Serving domain mode or Subdirectory publishing mode.
+
+#### STEP 1 - Serving domain
+
+As a first step, you need to provide the domain name on which the translated website is to be published.
+The proxy will only publish a project on one domain name at a time, the question is which one should be the primary - naked domain (example.com) or subdomain (www.example.com)? Once that question is decided, you should then set up server-side redirection to the "main" domain (best practice prefers the `HTTP301 MOVED PERMANENTLY` status code with the `Location` header).
+
+#### STEP 2 - Publishing mode
+
+##### Serving domain mode (fr.example.com)
+
+In the serving domain mode, the translation proxy will publish the translated site either on a subdomain of the original (the default behavior, such as fr.example.com), or on a completely separate naked domain (such as example.com). In order to use this mode, you (or the client) will have to modify the DNS settings corresponding to the original domain - the three to five records (three for subdomains, five for new naked domains) that need to be inserted in your DNS settings are found under the _Verification_ menu. These records will change as you enter or change the desired serving domain in the first step called _Serving domain_.
+
+##### Subdirectory publishing (example.com/fr)
+
+The alternative to subdomain-based publishing is to retain your own domain and publish the site as a subdirectory. I.e. the translated pages will appear under separate paths under the same domain as the one the project was created for (the original domain).
+
+Due to the way the proxy works, this requires a reverse proxy configuration to be placed in front of the web server. A variety of load balancer/reverse proxy solutions are available on the market, with `nginx`, `CloudFlare` and `AWS CloudFront` being three of the most well-known solutions available. See the vendor documentation for the details of setting up a reverse proxy (do note that nowadays, reverse proxies are monumentally powerful network solutions, and discussion of all their features is beyond the scope of this introductory description).
```

Comment:
On the other hand, these aren't really _code_, so a different highlighting should be used here...
Maybe _italics_ or **bold**?